entity.go

```diff
@@ -434,7 +434,7 @@ func (e *EntityDefinition) IsCompatible(e2 *EntityDefinition) error {
 			}
 
 			if !reflect.DeepEqual(index1, index2) {
-				return errors.Errorf("index mismatch: (%v vs %v)", index1, index2)
+				return errors.Errorf("index %s mismatch: (%v vs %v)", name, index1, index2)
```

Comment:
The error msg will be a little nicer if you use %q instead of %s

Reply:
done.